apps/wrf/wrf.py

```diff
@@ -4,12 +4,9 @@
 import os
 import re
 import string
-import sys
 import urllib
 import reframe as rfm
 import reframe.utility.sanity as sn
-
-sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..'))
```

Comment:
this is not needed and you could use relative imports as
```
from ...modules.utils import SpackTest
```
But it's probably cleaner using absolute imports instead.